superchain/superchain.go

```diff
@@ -479,6 +479,13 @@ func canonicalizeSemver(version string) string {
 	return version
 }
 
+type ContractBytecodeImmutables struct {
+	AnchorStateRegistry string `toml:"anchor_state_registry,omitempty"`
+	DelayedWETH         string `toml:"delayed_weth,omitempty"`
+	FaultDisputeGame    string `toml:"fault_dispute_game,omitempty"`
+	MIPS                string `toml:"mips,omitempty"`
```

Comment:
Perhaps add a comment here explaining the immutable info is encoded as a raw stringified JSON string but stored in a toml file (to make maintenance easier as well as readability).

Reply:
Done.